src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const сalculator = {
+    result: 0,
+
+    add(result, value) {
+      return result + value;
+    },
+
+    subtract(result, value) {
+      return result - value;
+    },
+
+    divide(result, value) {
+      return result / value;
+    },
+
+    multiply(result, value) {
+      return result * value;
+    },
+
+    operate(callback, value) {
```

Comment:
Consider adding a type check for the function and throwing an error if needed

Reply:
Haven't learned how to throw an error yet.